torch_xla/csrc/ops/ops.cpp

```diff
@@ -809,6 +810,22 @@ NodePtr Lerp(const Value& start, const Value& end, const Value& weight) {
   return start + weight * (end - start);
 }
 
+NodePtr Linspace(const Value& start, const Value& end, const int64_t steps) {
```

Comment:
This will work, the only "issue" is that it is not creating a Node call `Linspace`, it is returning a combination of other IR nodes. When user dump the IR graph he might not realized this `Put` IR was a result of `Linspace` op.

This for the most part is fine, and we do this a lot in early days to speed up the op lowering process, but we are trying to move away from it for the better debuggility. Could you create a `Linspace` op under ops dir like [kth_value](https://github.com/pytorch/xla/blob/master/torch_xla/csrc/ops/kth_value.h) and add a lower function(which prettry much follow the same logic you have here, you might want to check how `Put` is lowered and call that lower function directly).

I think one of the reason you lower this way is to make use of the `Arange` Node. I think it is a good time for us to actually refactor that a bit too. We can refacotr https://github.com/pytorch/xla/blob/master/torch_xla/csrc/ops/ops.cpp#L489-L550 into a helper function to return a `xla::Literal` and use that helper function in both `Arange` and `Linspace`